src/index.html

```diff
@@ -11,7 +11,141 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- Pesronal information group -->
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <label>
+        <fieldset class="personal_fieldset">
+          <legend>Pesronal information</legend>
+          <label>Surname:
+            <input
+            type="text"
+            autocomplete="off"
+            class="surname_input"
+            >
+          </label>
```

Comment:
```suggestion
          <label>
            Surname:
            <input
              type="text"
              autocomplete="off"
              class="surname_input"
            >
          </label>
```
also fix in other places